.github/workflows/rpm_build.yml

```diff
@@ -22,6 +22,7 @@ jobs:
           dnf install -y rpm-build rpmdevtools git make
           dnf module -y install go-toolset 
           rpmdev-setuptree
+          echo $GITHUB_SHA | cut -c1-8 > .commit
```

Comment:
Add .commit to a .gitignore?

Reply:
This is a GitHub workflow. .commit isn't created anywhere else. Why do you think it needs to be in .gitignore?